### ISSUE TITLE
TOC text is invisible unless hovered on

### ISSUE THREAD

**Issue by martin-martin** [mkdocs/mkdocs-bootstrap#33]
I tested the theme with a minimal setup, here's my `mkdocs.yaml` file:

```yaml
site_name: Test
site_description: Test Bootstrap Theme
site_author: Anonymous
theme:
  name: bootstrap
  shortcuts:
    help: 191    # ?
    next: 78     # n
    previous: 80 # p
    search: 83   # s
```

The resulting navigation of the theme is hard to use because the text is blue on blue:

![bootstrap](https://user-images.githubusercontent.com/12371494/97070785-5f8c8d80-15db-11eb-9d6c-ded5b9d09834.gif)

Any ideas why that might be happening? Thanks!

**Comment by machow** [comment]
The wheel uploaded to PyPI has an extra css file in it...

* `css/bootstrap-3.3.7.min.css`

@waylan is there any chance you can publish a patch to PyPI with a clean build?

**Comment by waylan** [comment]
This appears to be a duplicate which was fixed in version 1.1.1.
